hyperspy/tests/io/test_dm3.py

```diff
@@ -67,8 +67,8 @@ def test_get_time(self):
         assert self.imageobject._get_time("6:56:37 pm") == "18:56:37"
 
     def test_parse_string(self):
-        assert self.imageobject._parse_string("") is None
-        assert self.imageobject._parse_string("string") is "string"
+        assert self.imageobject._parse_string("") == None
```

Comment:
This one got caught up by accident, but I don't think it is any less clear like this.

Reply:
`_parse_string` is return a `None` type so the correct test is with `is`.

Reply:
Ah, yes. I searched to be sure - [PEP8 ](https://www.python.org/dev/peps/pep-0008/)is in agreement with you.